manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsService.java

```diff
@@ -7,4 +7,6 @@ public interface ConnectorsService {
     void createConnectorEntity(Processor processor);
 
     void deleteConnectorEntity(Processor processor);
+
+    void updateConnectorEntityDefinition(Processor processor);
```

Comment:
Nitpick: For consistency across the methods of this class, I think this should be named `updateConnectorEntity`

Reply:
Done.